sql/parser/sql.go

```diff
@@ -333,313 +333,319 @@ var yyExca = []int{
 	-2, 0,
```

Comment:
any chance of being able to introduce a generic comment line at the top of this file so that Github omits the diff?